bench/bench.js

```diff
@@ -10,7 +10,7 @@ function benchmark(name, setup, action) {
 
   // Warm up the JIT.
   var start = Date.now();
-  while ((Date.now() - start) < 10000 /* 10 seconds */) {
+  while ((Date.now() - start) < 5000 /* 5 seconds */) {
```

Comment:
This seems like it would be race condition prone, since the results are timing dependent. Is there a better way to warm up the JIT, that does not suffer from timing effects?